money_notebook_v141.py

```diff
@@ -417,6 +497,17 @@ def total_cal(s,e):
             elif len(uinput_split) == 2:
                 total_cal(uinput_split[1],uinput_split[1])
         else:print("【无效输入（错误代码14）】")
-        show_m = False
-    elif uinput == "f":break
+    elif re.search("\+", uinput) and len(uinput) == 1:

```

Comment:
move all these conditions into function. for example:
~~~python
def isAddPlan(uinput) {
    return re.search("\+", uinput) and len(uinput) == 1
}

elif isAddPlan(uinput):
    //blahblah
~~~